pycode/memilio-epidata/memilio/progress_indicator.py

```diff
@@ -0,0 +1,326 @@
+#############################################################################
```

Comment:
The progress indicator is more of a general utility, and does not depend on epdidata at all. This is why I put it above epidata.

Should we maybe create a folder pycode/utils? I think this would be more appropriate, but I don't know if it works well with the setup/installation.

We can also simply move the file into epidata - it is (currently) not used anywhere else.